modules/codegen/src/smithy4s/codegen/SmithyToIR.scala

```diff
@@ -607,7 +630,7 @@ private[codegen] class SmithyToIR(model: Model, namespace: String) {
           enumDef.getName().asScala
         )
       // List
-      case (N.ArrayNode(list), Type.List(mem)) =>
+      case (N.ArrayNode(list), Type.List(mem, _)) => // todo sure ?
```

Comment:
This is just ignoring the hints because you don't need them here? That seems fine to me